docs/docs/reference/config-managers/k8s/crd.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Configuration Managers
+id: configfile
+layout: docs
+description: Secretless Broker Documentation
+permalink: docs/reference/config-managers/k8s/crd.html
+---
+
+## Kubernetes Custom Resource Definition (CRD)
+The Kubernetes CRD plugin (`k8s/crd`) plugin allows the use of Kubernetes-specific
+custom resource definition to trigger and specify the configuration for Secretless Broker.
```

Comment:
"of a k8s-specific CRD" or "of k8s-specific CRDs" - there's a plurality mismatch here